src/components/structures/TimelinePanel.tsx

```diff
@@ -294,6 +320,7 @@ class TimelinePanel extends React.Component<IProps, IState> {
         cli.on(MatrixEventEvent.Decrypted, this.onEventDecrypted);
         cli.on(MatrixEventEvent.Replaced, this.onEventReplaced);
         cli.on(ClientEvent.Sync, this.onSync);
+        cli.on(ClientEvent.DumpDebugLogs, this.onDumpDebugLogs);
```

Comment:
Is there some better place to fire and listen to events from for this kind of thing? This doesn't need to live in `matrix-js-sdk` at all technically although maybe could be extended to also log some info from components in `matrix-js-sdk`. Don't want to optimize for a future case that will never happen though.

Reply:
This is perfect! 👌